File: Projeto Cobra/src/cobrafxml/db/cobradb/db/entidades/Estado.java
package cobrafxml.db.cobradb.db.entidades;

public class Estado
{
    private int id;
    private String nome, sigla;

    public Estado() {
        this(0, "", "");
    }
    
    public Estado(String nome, String sigla) {
        this(0, nome, sigla);
    }
    
    public Estado(int id, String nome, String sigla) {
        this.id = id;
        this.nome = nome;
        this.sigla = sigla;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getSigla() {
        return sigla;
    }

    public void setSigla(String sigla) {
        this.sigla = sigla;
    }

    @Override
    public String toString() {
        return nome;
    }
}

File: DogSoft/DogSoft/Modelo/Estado.cs
using DogSoft.Persistencia;
using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DogSoft.Modelo
{
    public class Estado
    {
        private int codigo;
        private string sigla, nome;
        private EstadoBD estadoBD;
        public Estado()
        {

        }
        public Estado(int codigo, string sigla, string nome)
        {
            this.codigo = codigo;
            this.sigla = sigla;
            this.nome = nome;
        }
        public int getCodigo()
        {
            return codigo;
        }

        public void setCodigo(int codigo)
        {
            this.codigo = codigo;
        }

        public string getNome()
        {
            return nome;
        }

        public void setNome(string nome)
        {
            this.nome = nome;
        }

        public string getSigla()
        {
            return sigla;
        }

        public void setSigla(string sigla)
        {
            this.sigla = sigla;
        }
        public DataTable buscAll()
        {
            estadoBD = new EstadoBD(Banco.conecta());
            return estadoBD.buscAll();
        }

        public DataTable busc(int codigo)
        {
            estadoBD = new EstadoBD(Banco.conecta());
            return estadoBD.busc(codigo);
        }
    }
}
